src/Microsoft.DotNet.ProjectModel/ProjectSystem/ProjectSystem.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.Frameworks;
+
+namespace Microsoft.Extensions.ProjectModel.ProjectSystem
+{
+    public class ProjectSystem
```

Comment:
I think workspace is better. ProjectSystem doesn't mean anything. It's just a collection of projects.